### PR TITLE
Create danusesgithub.js member profile js file

### PR DESCRIPTION
**Pull request by DanUsesGitHub**
## Linked Issue

- #13

## Description

Added my user profile data

## Methodology

Followed instructions!

### CODE REVIEW

**Comment by DanUsesGitHub** [comment]
david s pumpkins

File: src/_data/members/members/danusesgithub.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  // github username: required
+  github: 'danusesgithub',
+  //
+  // everything below here is optional. by default, we pull most profile data from your github profile.
+  // you can override that data here, as well as provide some additional account links below
+  //
+  // name - if not defined here, will default to your Name on github if defined, if not, then your username
+  name: 'Not Dan Ott',
```
Comment (danieltott) [inline src/_data/members/members/danusesgithub.js:9]: This is not your real name

**Review by danieltott — CHANGES_REQUESTED** [review]
Fix this!!!!!!!

File: src/_data/members/members/danusesgithub.js (hunk 1 above)
Comment (DanUsesGitHub) [reply src/_data/members/members/danusesgithub.js:9]: Yes it is!!

Comment (DanUsesGitHub) [reply src/_data/members/members/danusesgithub.js:9]: OK fixed

**Pull request closed without merging**